helm-charts/mach-operator/values.yaml

```diff
@@ -0,0 +1,239 @@
+    # Optional, default value: 1000ms
+    interval_ms = 4000
+
+    ##############################################
+    # Mach scheduler configuration               #
+    ##############################################
+    [scheduler]
+    # The timeout waiting for verify result.
+    # If it times out, the verify result will be checked again in the next tick.
+    # Optional, default value: 100ms
+    verify_timeout_ms = 100
+
+    ##############################################
+    # Mach prometheus configuration              #
+    ##############################################
+    [prometheus]
+    # The TCP port of Prometheus exporter
+    # Optional, default value: 9094
+    port = 9094
+    # The HTTP URI path for fetching Prometheus metrics
+    # Optional, default value: "/metrics"
+    metrics_path = "/metrics"
+    # The indication if expose Prometheus exporter on all interfaces
+    # true: listen address = "0.0.0.0:port"
+    # false: listen address = "127.0.0.1:port"
+    # Optional, default value: false
+    external = true
+node:
+  image:
+    repository: public.ecr.aws/altlayer/mach-operator
+    pullPolicy: IfNotPresent
+    tag: "latest"
+  resources:
+    limits:
+      cpu: 4
+      memory: 4Gi
+    requests:
+      cpu: 2
+      memory: 4Gi
+  args:
+    - --config=/app/config/operator.yaml
+  env:
+    - name: OPERATOR_BLS_KEY_PASSWORD
+      value: ""
+  readinessProbe:
+    httpGet:
+      path: /metrics
+      port: 9090
+    initialDelaySeconds: 30
+    periodSeconds: 10
+    timeoutSeconds: 3
+    failureThreshold: 3
+    successThreshold: 1
+  livenessProbe:
+    httpGet:
+      path: /metrics
+      port: 9090
+    initialDelaySeconds: 1200
+    periodSeconds: 30
+    timeoutSeconds: 3
+    failureThreshold: 2
+    successThreshold: 1
+
+verifier:
+  resources:
+    limits:
+      cpu: 4
+      memory: 4Gi
+    requests:
+      cpu: 2
+      memory: 4Gi
+  image:
+    repository: public.ecr.aws/altlayer/mach
+    pullPolicy: IfNotPresent
+    tag: "latest"
+  args:
+    - --config=/app/mach-config.toml
+    - --genesis-config=/app/genesis.json
+  env:
+    - name: RUST_LOG
+      value: "info"
+  readinessProbe:
+    httpGet:
+      path: /metrics
+      port: 9094
+    initialDelaySeconds: 30
+    periodSeconds: 10
+    timeoutSeconds: 3
+    failureThreshold: 3
+    successThreshold: 1
+  livenessProbe:
+    httpGet:
+      path: /metrics
+      port: 9094
+    initialDelaySeconds: 1200
+    periodSeconds: 30
+    timeoutSeconds: 3
+    failureThreshold: 2
+    successThreshold: 1
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
```

Comment:
We can change the name here, but it isn't reflected in `statefulset.yaml` and `serviceaccount.yaml`, need to change them to support it